src/Moryx.Runtime.Kernel/KernelServiceCollectionExtensions.cs

```diff
@@ -87,6 +93,19 @@ public static IConfigManager UseMoryxConfigurations(this IServiceProvider servic
             return configManager;
         }
 
+        /// <summary>
+        /// Use moryx file system and configure base directory
+        /// </summary>
+        /// <returns></returns>
+        public static IMoryxFileSystem UseMoryxFileSystem(this IServiceProvider serviceProvider, string path)
```

Comment:
Should this have a default path which points to a directory next to, not within the apps directory?